### DISCUSSION TITLE
AttributeError: module 'ultralytics.solutions' has no attribute 'ObjectCounter'

### DISCUSSION THREAD

**Discussion by mickycooky** [ultralytics/ultralytics#14763] [Q&A]
hi, im university student from japan
i am studying about human detection 

i base on https://docs.ultralytics.com/ja/guides/object-counting/#real-world-applications"
but on "#init Object Counter", displayed "AttributeError: module 'ultralytics.solutions' has no attribute 'ObjectCounter"

also i base on https://github.com/orgs/ultralytics/discussions/6738
but displayed "No such file or directory" maybe setup.py

my dev? is probably no problem
im not good at English and programming, but thank you

**Comment by pderrenger** [comment]
@mickycooky thank you for reaching out. It seems like you are encountering an issue with the `ObjectCounter` class. Please ensure you are using the latest version of the Ultralytics package, as updates often resolve such issues. You can update the package using `pip install --upgrade ultralytics`.

If the problem persists, it might be due to a missing or incorrect import. Verify that you are importing the `ObjectCounter` class correctly from the `ultralytics.solutions` module. If you continue to face issues, please provide more details about your setup and the exact steps you are following, so we can assist you further.

For more detailed guidance, you can refer to the [Ultralytics Object Counter documentation](https://docs.ultralytics.com/reference/solutions/object_counter/). Thank you for your understanding.

**Comment by cobaltautomationdev** [comment]
HI, same error as me, my OS is windows10, but in google colab it works.

**Reply by cobaltautomationdev** [reply]
just update to latest ultralytics version 8.2.78 today , it works in in windows, you can try.

**Reply by glenn-jocher** [reply]
@cobaltautomationdev please ensure you're using the latest version of Ultralytics by updating with `pip install ultralytics --upgrade`. This should resolve the issue.
